firefox_ui_tests/remote/security/test_safe_browsing_notification.py

```diff
@@ -0,0 +1,150 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import time
+
+from marionette import By
+from marionette.errors import (
+    NoSuchElementException,
+    StaleElementException
+)
+
+from firefox_ui_harness.testcase import FirefoxTestCase
+
+
+class TestSafeBrowsingNotificationBar(FirefoxTestCase):
+
+    def setUp(self):
+        FirefoxTestCase.setUp(self)
+
+        self.urls = [
+            # Phishing URL object = mozmill 18Feb2015
+            ("safebrowsing.notAForgeryButton.label",
+             "www.google.com/safebrowsing/report_error",
+             "https://www.itisatrap.org/firefox/its-a-trap.html"),
+            # Malware URL object  = mozmill 18Feb2015
+            ("safebrowsing.notAnAttackButton.label",
+             "www.stopbadware.org",
+             "https://www.itisatrap.org/firefox/its-an-attack.html"),
+        ]
```

Comment:
Those are not all URLs. I would suggest that in this case you name the variable `self.test_data`, and keep the dict and not use an array.